src/borg/archiver.py

```diff
@@ -2412,10 +2412,31 @@ def do_break_lock(self, args, repository):
                 # susan is a nice person
                 # include susans home
                 + /home/susan
+                # also back up this exact file
+                + pf:/home/bobby/specialfile.txt
                 # don't backup the other home directories
                 - /home/*
                 # don't even look in /proc
-                ! /proc\n\n''')
+                ! /proc
+
+            As another example, you could do the same thing in either of these ways::
+
+                # specified on command line
+                borg create --exclude /home/bobby/junk repo::arch /home/bobby /home/susan
+                # specified in patternfile
+                borg create --patterns-from patternfile.lst repo::arch
+
+            The patternfile::
+
+                # note that excludes use fm: by default and patternfiles use sh: by default.
+                # therefore, we need to specify fm: to have the same exact behavior.
+                P fm
+                R /home/bobby
+                R /home/susan
+
+                - /home/bobby/junk
+
+            This allows you to share the same patterns between multiple repositories without needing to specify them on the command line.\n\n''')
```

Comment:
borg can output this help directly on the terminal.
if a line is longer than 80 chars, rather continue on next line.